buddyboss/buddyboss-platform
# PROD-750 #2877 - Fix: Delete & Select all icons shows up when there is no video

https://buddyboss.atlassian.net/browse/PROD-750

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2877 .

### How to test the changes in this Pull Request:

1. Go to Video (Click on Profile icon > Video)
2. Can see, the delete and select all buttons do not appear if no video

### Proof Screenshots or Video
https://vimeo.com/632748765/b8d1291bb3


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
